logutils/requestlog/request_log.go

```diff
@@ -0,0 +1,59 @@
+package requestlog
+
+import (
+	"errors"
+	"sync/atomic"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/status-im/status-go/logutils"
+)
+
+var (
+	// requestLogger is the request logger object
+	requestLogger log.Logger
+	// isRequestLoggingEnabled controls whether request logging is enabled
+	isRequestLoggingEnabled uint32
```

Comment:
Just use `atomic.Bool`, it has a proper interface 🙌 
```suggestion
	isRequestLoggingEnabled atomic.Bool
```